deploy/hypershift-cluster-monitoring-config/cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hshift-cluster-monitoring-config
```

Comment:
```suggestion
  name: hypershift-cluster-monitoring-config
```

Reply:
@supreeth7 The configmap name should be same the the classic one, isn't it?

Reply:
Yes, that's right, apparently CMO defaults to recognizing configmap names "cluster-monitoring-config" only.
```suggestion
  name: cluster-monitoring-config
```